third-party/realdds/src/dds-stream-server.cpp

```diff
@@ -158,18 +174,30 @@ void dds_stream_server::stop_streaming()
     _image_header.invalidate();
 }
 
+void dds_stream_server::close()
+{
+    _writer.reset();
```

Comment:
Should check if streaming.

Reply:
And throw if not? Not sure...

Reply:
Then it can be `is_streaming()` while `! is_open()`

Reply:
I think that we should check `is_streaming` and throw exception or at least call `stop_streaming` before reset.
Otherwise the device will show a state of streaming while not open.

Reply:
We said we're removing `is_streaming` and all the related.